provisioning/WFGP-292_doc_galleon-pack.adoc

```diff
@@ -0,0 +1,164 @@
+---
+categories:
+- wf-galleon
+- maven
+- user-experience
+stability-level: community
+issue: https://github.com/wildfly/wildfly-proposals/issues/738
+feature-team:
+ developer: jmesnil
+ sme:
+  - jfdenise
+  - ehsavoie
+ outside-perspective:
+  - yersan
+---
+= Generate documentation for Galleon feature packs
+:author:            Jeff Mesnil
+:email:             jmesnil@ibm.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+*_"Javadoc" for feature packs_*
+
+The goal of that proposal is to provide documentation of the capabilities and resources that are provided by WildFly and its ecosystem of Galleon feature packs.
+This documentation is self-contained and automatically generated when the feature packs are built.
+
+== Overview
+
+At its core, WildFly is composed of extensions that provide management resources (grouped in subsystems and deployment resources).
+WildFly is providing a model reference for these resources at https://docs.wildfly.org/36/wildscribe/.
```

Comment:
WDYT about starting to call this "management API reference" not "model reference"?

We've used the term "model" around this general area for years but I couldn't tell you precisely what we mean by it. And I suspect a user has no clue.